napari/_qt/widgets/qt_scrollbar.py

```diff
@@ -17,13 +17,17 @@ class ModifiedScrollBar(QScrollBar):
     def _move_to_mouse_position(self, event):
         opt = QStyleOptionSlider()
         self.initStyleOption(opt)
+
+        point = (
```

Comment:
See similar comment in `_qt_main_window.py` - I can't link to it before submitting the review.